neuralpp/test/quick_tests/symbolic/interpreter_test.py

```diff
@@ -62,15 +62,18 @@ def boolean_function_of_arity(arity: int) -> BasicConstant:
 
 
 def dict_to_sympy_context(kv_map: dict) -> SymPyExpression:
-    result = sympy.S.true
+    result = True
```

Comment:
The name `result` suggests this is the thing to be returned, but that doesn't seem to be true. Should it really be called `result`?